explanations/python-imports-check-typo.md

```diff
@@ -0,0 +1,16 @@
+Not all python packages include tests. However it should be possible, at a minimum, to "smoke test" every python package my simply checking that the modules it declares can be `import`ed.
+
+For example, `pythonImportsCheck` can be used like this:
+```
+buildPythonPackage {
+  name = "mymodule";
+  src = ...
+
+  pythonImportsCheck = [
+    "mymodule.submodule1"
```

Comment:
If there is a higher level namespace like mymodule it is not required to import all sub-namespaces if they are already imported.